README.md

```diff
@@ -189,12 +189,14 @@ $ git clone git@github.com:nirname/swarm.git $GOPATH/src/github.com/ethersphere/
 
 ### Vendored Dependencies
 
-All dependencies are tracked in the `vendor` directory. We use `go mod` to manage depenedencies, except for `go mod vendor` command. Vendoring is done by Makefile rule `make vendor` which uses `go mod vendor` and additionally copies cgo dependencies into `vendor` directory from go modules cache.
+All dependencies are tracked in the `vendor` directory. We use `go mod` to manage dependencies, except for `go mod vendor` command. Vendoring is done by Makefile rule `make vendor` which uses `go mod vendor` and additionally copies cgo dependencies into `vendor` directory from go modules cache.
```

Comment:
I think I would remove the first sentence:

> All dependencies are tracked in the `vendor` directory. We use `go mod` to manage dependencies, except for `go mod vendor` command. 

The second sentence already contains that info.

Reply:
Yes, that is right. Removed.